pulp_container/app/registry.py

```diff
@@ -222,7 +222,7 @@ async def dispatch_tag(tag, response_headers):
 
         """
         try:
-            artifact = tag._artifacts.get()
+            artifact = tag.tagged_manifest._artifacts.get()
```

Comment:
@dkliban do you remember why we decided to have Tag as a content with artifact? plus that artifact is shared between a tag and manifest? We could have just had Tag without artifact. Thoughts?

Reply:
I don't remember why, but I don't think we need to do that. Also, I now realize that this bug that I discovered only occurs because i am not associating any Artifact with the tag during 'push'.